db/migrate/20220202002459_create_rule_groups.rb

```diff
@@ -0,0 +1,24 @@
+class CreateRuleGroups < ActiveRecord::Migration[6.1]
+  PHONY_BENCHMARK = Xccdf::Benchmark.find_or_create_by!(
+    ref_id: 'phony_ref_id',
+    version: 'v0.0.0',
+    title: 'phony title',
+    description: 'phony description'
+  )
+
+  def change
+    create_table :rule_groups, id: :uuid do |t|
+      t.string :ref_id
+      t.string :title
+      t.text :description
+      t.text :rationale
+      t.string :parent_type
+      t.string :ancestry
+
+      t.timestamps
+    end
+    add_index :rule_groups, :ancestry
+    add_column :rule_groups, :benchmark_id, :uuid, null: false,
+      default: PHONY_BENCHMARK.id
```

Comment:
Is this really necessary?

Reply:
I'm not sure really.. Profile and Rule has it set up like this but I couldn't figure out why.. 
@vkrizan Do you know if setting the default is necessary for the `benchmark_id` on `rule_groups`?

Reply:
I'm trying to remember why it was added there... It could have been there to avoid some situation in testing. But, I think that it is no longer relevant since the introduction of `faker` that generates correct objects.